benchmark-scripts/smoke_test.sh

```diff
@@ -8,26 +8,49 @@
 # Download media
 ./download_sample_videos.sh
 
+PLATFORM=$1
+CPU_ONLY=$2
+
+if [ "$PLATFORM" == "" ]
+then
+    PLATFORM="core"
+fi
+
+if [ "$CPU_ONLY" == "" ]
+then
+    CPU_ONLY=0
+fi
+
 sudo rm -rf results || true
-sudo rm -rf smoke_test_camera_simulator_full/  || true
-sudo rm -rf smoke_test_camera_simulator_yolov5/  || true
-sudo rm -rf smoke_test_file_full/ || true
-sudo rm -rf smoke_test_stream_density/ || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_full/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_yolov5_classification/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_yolov5/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_file_full/ || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_stream_density/ || true
```

Comment:
suggest changing the file name with explicitly spelling out the platform and cpu only:
like :
```suggestion
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_full/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_yolov5_classification/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_yolov5/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_file_full/ || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_stream_density/ || true
```

Reply:
I like it! will merge your suggestion